app/src/main/org/runnerup/widget/TitleSpinner.java

```diff
@@ -165,4 +166,11 @@ public void setOnSetValueListener(SpinnerInterface.OnSetValueListener listener)
   public void setOnCloseDialogListener(SpinnerInterface.OnCloseDialogListener listener) {
     mPresenter.setOnCloseDialogListener(listener);
   }
+
+  // Instead of android:entries="@array/anArray"
+  public void setArrayEntries(String[] entries) {
+    var adapter = new ArrayAdapter<CharSequence>(
+        getContext(), android.R.layout.simple_spinner_item, entries);
+    mSpinner.setAdapter(adapter);
```

Comment:
```suggestion
    adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
    mSpinner.setAdapter(adapter);
```

The left image is without the call to `setDropDownViewResource`. 

<img width="770" height="334" alt="Image" src="https://github.com/user-attachments/assets/f6ee2a7e-bc07-4ff7-a691-e9f97e99c811" />